kitsune/questions/views.py

```diff
@@ -617,9 +631,14 @@ def aaq_step2(request, product_key):
     return aaq(request, product_key=product_key, step=2)
 
 
-@login_required
 def aaq_step3(request, product_key, category_key=None):
     """Step 3: Show full question form."""
+
+    # Since removing the @login_required decorator for MA form
```

Comment:
We should supply a `next` url here so that users can continue from where they left the process. I believe the `login_required` decorator uses the `redirect_to_login` method from `django.contrib.auth.views` which accepts the `next` url and the login url as params

Reply:
I think I've fulfilled this.